tests/performance-tests/src/performance/load_test.go

```diff
@@ -0,0 +1,89 @@
+package performance_test
+
+import (
+	"bufio"
+	"os"
+	"path/filepath"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gexec"
+	"github.com/tsenart/vegeta/lib"
+
+	"github.com/cloudfoundry-incubator/cf-test-helpers/cf"
+	"github.com/cloudfoundry-incubator/cf-test-helpers/generator"
+	"github.com/cloudfoundry-incubator/cf-test-helpers/helpers"
+)
+
+const (
+	loadTestRate     = 100
+	loadTestDuration = 10 * time.Second
+	loadTestLatency  = 300
```

Comment:
From meatspace: @combor is going to add a unit to this (`* time.Millisecond` we think) so that it's easier to understand the value and we can remove the `/ 1000000` from the assertions further down.